splinepy/io/json.py

```diff
@@ -0,0 +1,153 @@
+"""
+Export splines in custom json format
+"""
+
+import base64
+import json
+import logging
+import numpy as np
+
+def load(fname):
+    """
+    Loads splines from json file. Returns dict-splines
+
+    Parameters
+    -----------
+    fname: str
+
+    Returns
+    --------
+    spline_list: list
+    """
+    # Import data from file into dict format
+    jsonbz = json.load(open(fname, "r"))
+
+    spline_list = []
+    base64encoding = jsonbz["Base64Encoding"]
+
+    # Init return type
```

Comment:
This is nice. But I think then we should match the return style like this for all io